### PR TITLE
kraken: line_section doesn't disrupt journey not stopping on the section

### PR DESCRIPTION
**Pull request by kinnou02**
https://jira.kisio.org/browse/NAVITIAII-2495

### CODE REVIEW

**Review by xlqian — APPROVED** [review]

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -959,6 +959,17 @@ It will retrieve all the journeys from the resource (in order to make *[isochron
 
 The [isochrones](#isochrones) service exposes another response structure, which is simpler, for the same data.
 
+### <a name="journeys-disruptions"></a> Disruptions
+By default Navitia compute journeys that doesn't take into account the disruptions, these will only be reported into the response.
+If you want to provide journeys without blocking disruptions you need to use the parameter `data_freshness=realtime`.
+
+In a journeys response there is disruptions in a lot of places, and not all have the same meaning.
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: * there **are** disruptions

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -959,6 +959,17 @@ It will retrieve all the journeys from the resource (in order to make *[isochron
...
+Each journey has a `status` attribute that indicate the most serious disruptions effect on this journey.
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: indicate**s**

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -959,6 +959,17 @@ It will retrieve all the journeys from the resource (in order to make *[isochron
...
+The disruptions are on the sections, the ones that impact the journeys are in the links of the display_informations of the sections (`sections[].display_informations.links[]`)
+
+There also some others disruptions on the responses, these doesn't impact directly the journey but might affect them, by example some intermediate stops of a sections can be
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: - These  **don't**

- **for** example

Comment (TeXitoi) [reply documentation/slate/source/includes/apis.md]: There **are** also some

**Review by nonifier — APPROVED** [review]

**Pull request merged**